tables/articles.csv

```diff
@@ -1,403 +1,404 @@
-Mendes-Correa21,10.1101/2021.06.11.21257717,NULL,"Mendes-Correa, MC",2021,2021-12-06,NULL
-Pommeret21,10.1016/j.annonc.2021.07.015,NULL,"Pommeret, F",2021,2021-12-07,NULL
-Monrad21,10.1093/ofid/ofab295,NULL,"Monrad, I",2021,2021-12-07,NULL
-Sabin21,10.1101/2021.10.02.21264415,NULL,"Sabin, A",2021,2021-12-07,NULL
-Sepulcri21,10.1093/ofid/ofab217,NULL,"Sepulcri, C",2021,2021-12-07,NULL
-Cele21c,10.1101/2021.12.08.21267417,NULL,"Cele, S",2021,2021-12-07,2021-12-14
-Wilhelm21b,10.1101/2021.12.07.21267432,NULL,"Wilhelm, A",2021,2021-12-08,NULL
-Cao21,10.1101/2021.12.07.470392,NULL,"Cao, YR",2021,2021-12-10,NULL
-Zhang21g,10.1080/22221751.2021.2017757,NULL,"Zhang, L",2021,2021-12-13,NULL
-Roessler21,10.1101/2021.12.08.21267491,NULL,"Roessler, A",2021,2021-12-13,NULL
-Dejnirattisai21b,10.1101/2021.12.10.21267534,NULL,"Dejnirattisai, W",2021,2021-12-13,NULL
-Nemet21,10.1101/2021.12.13.21267670,NULL,"Nemet, I",2021,2021-12-15,NULL
-Zhang21f,10.1101/2021.05.15.21257254,NULL,"Zhang, X",2021,2021-12-15,NULL
-Planas21c,10.1101/2021.12.14.472630,NULL,"Planas, D",2021,2021-12-15,NULL
-Doria-Rose21,10.1101/2021.12.15.21267805,NULL,"Doria-Rose, NA",2021,2021-12-16,NULL
-Aggarwal21,10.1101/2021.12.14.21267772,NULL,"Aggarwal, A",2021,2021-12-16,NULL
-Cameroni21,10.1101/2021.12.12.472269,NULL,"Cameroni, E",2021,2021-12-17,NULL
-Liu21l,10.1101/2021.12.14.472719,NULL,"Liu, L",2021,2021-12-17,NULL
-Schmidt21b,10.1101/2021.12.12.21267646,NULL,"Schmidt, F",2021,2021-12-20,NULL
-VanBlargan21,10.1101/2021.12.15.472828,NULL,"VanBlargan, LA",2021,2021-12-22,NULL
-AstraZenica21c,NULL,https://opendata.ncats.nih.gov/variant/datasets,"AstraZenica",2021,2021-12-24,NULL
-Lu21b,10.1093/cid/ciab1041,NULL,"Lu, L",2021,2021-12-24,NULL
-Yu21c,10.1101/2021.12.17.21267961,NULL,"Yu, X",2021,2021-12-25,NULL
-Changrob21,10.1128/mBio.02975-21,NULL,"Changrob, S",2021,2021-12-25,NULL
-Zeng21b,10.1101/2021.12.16.472934,NULL,"Zeng, C",2021,2021-12-25,NULL
-Dolzhikova21,10.1101/2021.12.17.21267976,NULL,"Dolzhikova, IV",2021,2021-12-25,NULL
-Edara21d,10.1101/2021.12.20.473557,NULL,"Edara, VV",2021,2021-12-25,NULL
-Zou21b,10.1101/2021.12.20.473584,NULL,"Zou, J",2021,2021-12-27,NULL
-Ai21,10.1080/22221751.2021.2022440,NULL,"Ai, J",2021,2021-12-27,NULL
-Muik21b,10.1101/2021.12.22.21268103,NULL,"Muik, A",2021,2021-12-28,NULL
-Lusvarghi21,10.1101/2021.12.22.473880,NULL,"Lusvarghi, S",2021,2021-12-29,NULL
-Dejnirattisai21c,10.1101/2021.12.03.471045,NULL,"Dejnirattisai, W",2021,2021-12-29,NULL
-Hoffmann21d,10.1101/2021.12.12.472286,NULL,"Hoffmann, M",2021,2021-12-29,NULL
-Arien21,10.1101/2021.12.23.21268316,NULL,"Arien, KK",2021,2022-01-03,NULL
-Sheward21,10.1101/2021.12.19.473354,NULL,"Sheward, DJ",2021,2022-01-03,NULL
-Syed21,10.1101/2021.12.20.21268048,NULL,"Syed, AM",2021,2022-01-03,NULL
-Wang21k,10.1101/2021.12.24.474138,NULL,"Wang, X",2021,2022-01-03,NULL
-Zhou21e,10.1101/2021.12.27.474307,NULL,"Zhou, T",2021,2022-01-04,NULL
+ref_name	doi	url	first_author	year	date_added	date_updated

```

Comment:
`tables/articles.csv` should be saved as "CSV UTF-8" format, instead it is saved as "Tab delimited text". In fact, all .csv files should be saved as "CSV UTF-8" format.